### PR TITLE
Fix some typos on SuggestionV2's README

### PR DESCRIPTION
**Pull request by MatiasMFM2001**
Some notes:

- In `Configuration Variables -> 4.` I've seen this, but I'm not sure if that's OK:
> This provides a good way to organise all approved suggestions. This provides a good way to organise all implemnted suggestions into a separate channel.

- In `Configuration Variables -> 7. and 8.` I've edited this:
> Supports both custom as well as inbuilt emojis, for which it is recommended...

Is that better than previous redaction?

### CODE REVIEW

**Comment by Satty9361** [comment]
The emoji one is incorrect. Unicode is only to be used for inbuilt emojis since unicode for custom emojis dont exist. You'd use the full emoji output after escaping it for those.

**Comment by MatiasMFM2001** [comment]
Oh

Original:
> The emoji for upvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters. Example : "👍".

Edited:
> The emoji for upvote reaction. Supports both custom as well as inbuilt emojis, for which it is recommended to use corresponding unicode characters. Example : "👍".

So, both are incorrect? Anyways, I would add your line (thanks!):
> For custom emojis, you'd use the full emoji output after escaping it for those.

**Comment by Jo3-L** [comment]
what's the status of this PR currently?

**Comment by MatiasMFM2001** [comment]
In my fork, `Configuration Variables -> 4` isn't modified. And `Configuration Variables -> 7. and 8.` is modified, but I'm not sure if it's incorrect (Satty's comment)

I see some changes in your repo were done. Should I sync them with mine?

File: suggestion/v2/README.md [hunk 1]
```diff
@@ -19,35 +19,35 @@ Example - For a server with prefix - `?`, Trigger would be : `(?i)\A(\?\s?|<@!?2
...
 
 6. `$Cooldown` -
 
-	Colldown in seconds between consucutive suggestions. This prevents suggestion misuse/spam. Can be set to 0 to disable. Mods/Admins can bypass the cooldown.
+	Cooldown in seconds between consecutive suggestions. This prevents suggestion misuse/spam. Can be set to 0 to disable. Mods/Admins can bypass the cooldown.
 
 7. `$Upvote` -
 
-	The emoji for upvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters. Example : "👍".
+	The emoji for upvote reaction. Supports both custom as well as inbuilt emojis, for which it is recommended to use corresponding unicode characters. Example : "👍".
```
Comment (Jo3-L) [inline suggestion/v2/README.md:46]: This (imo) actually makes the sentence less clear, please revert.

File: suggestion/v2/README.md [hunk 2]
```diff
@@ -19,35 +19,35 @@ Example - For a server with prefix - `?`, Trigger would be : `(?i)\A(\?\s?|<@!?2
...
 	
 8. `$Downvote` -
 
-	The emoji for downvote reaction. Supports both custom as well as inbuilt emojis. For inbuilt emojis, it is recommended to use corresponding unicode characters. Example : "👎".
+	The emoji for downvote reaction. Supports both custom as well as inbuilt emojis, for which it is recommended to use corresponding unicode characters. Example : "👎".
```
Comment (Jo3-L) [inline suggestion/v2/README.md:50]: Please revert this according to above comment as well.

**Review by Jo3-L — CHANGES_REQUESTED** [review]
Some nits

**Comment by Jo3-L** [comment]
thanks, lgtm.

**Comment by MatiasMFM2001** [comment]
You're welcome!

**Pull request merged**